book/doctrine.rst

```diff
@@ -722,6 +722,33 @@ instead of querying for rows on a table (e.g. ``product``).
 When querying in Doctrine, you have two options: writing pure Doctrine queries
 or using Doctrine's Query Builder.
 
+Querying for Objects with DQL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of using the ``QueryBuilder``, you can alternatively write the queries
```

Comment:
How about this. The first part is copied from the QueryBuilder section:

```rst
Imagine that you want to query for products, but only return products that
cost more than ``19.99``, ordered from cheapest to most expensive. You can use
Doctrine's native SQL-like language called DQL to do query for this::
```

Then, the first sentence under `Querying for Objects Using Doctrine's Query Builder` should change to be something like:

```rst
Instead of writing a DQL string, you can alternatively use a helpful object called
the ``QueryBuilder`` to build that string for you::
```